### ISSUE TITLE
dataset details

### ISSUE THREAD

**Issue by pawopawo** [TAMU-VITA/FasterSeg#14]
Why did you search directly in the train set and did not sample the val set from the cityscapes train set?
Will the searched arch not overfit in the cityscapes val set?

**Comment by chenwydj** [comment]
Hi @pawopawo !

What did you mean by "sample the val set from the cityscapes train set"?

We only leveraged the training set for the search, without using the validation set. We followed the same setting in [DARTS](https://github.com/quark0/darts/blob/master/cnn/train_search.py#L93).

**Comment by pawopawo** [comment]
![image](https://user-images.githubusercontent.com/38682321/75223951-83949680-57e2-11ea-87bd-3736424f6a71.png)
darts sample the val set from the train set.

**Comment by chenwydj** [comment]
DARTS split the training set half and half for optimizing w and alpha: https://github.com/quark0/darts/blob/master/cnn/train_search.py#L91

Same as what we did: https://github.com/TAMU-VITA/FasterSeg/blob/master/search/train_search.py#L108
